package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@metarhia/metagram",
+  "version": "1.0.0",
+  "description": "ERD generator for metaschema",
+  "main": "metagram.js",
```

Comment:
Add keys:
- files
- types

Reply:
I have added `files` but see no point in `types`.
1. It is not a library, it's a cli tool.
2. There are no types. If package grows bigger we can go on and add typings, but now I do not see a reason.

Reply:
If we will use it just as a cli tool it's ok, but in future I'd like to generate visualization in runtime from UI